.travis.yml

```diff
@@ -14,10 +14,10 @@ android:
     - platform-tools
 
     # The BuildTools version used by your project
-    - build-tools-23.0.3
+    - build-tools-30.0.3
```

Comment:
okay I will change it back to 30. I initially tried changing it to 30 to see if it would fix the build but it didn't change anything so I just reverted back to 23